components/HowItWorks/Accordion/AccordionCard/AccordionCard.tsx

```diff
@@ -6,18 +6,27 @@ import { useState } from "react";
 
 interface AccordionCardProps {
   data: AccordionInfoType;
-  onClick(): void;
+  alternateClass: "accordion" | "faqAccordion";
 }
 
-const AccordionCard = ({ data, onClick }: AccordionCardProps) => {
-  const { id, title, description, defaultChecked = false } = data;
+const AccordionCard = ({ data, alternateClass }: AccordionCardProps) => {
+  const { title, description, defaultChecked = false } = data;
   const [cardOpen, setCardOpen] = useState(defaultChecked);
   const handleCardClick = () => {
     setCardOpen(!cardOpen);
   };
-
+  const getAlternateClass = () => {
+    switch (alternateClass) {
+      case "accordion":
+        return "px-6 py-8 bg-gray-50 rounded-xl";
+      case "faqAccordion":
```

Comment:
@oscarcorellan Don't associate the place where this component is used. Instead use a name more general that describes the changes like: 
```suggestion
      case "whiteBg":
```